tests/cloudlaunchprocess/test.csv

```diff
@@ -0,0 +1,25 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+comment,0,,"Test 1 - Launch and then kill a process with all possible cmd line args"
+run,0,0,q,.sys.launch input1,,,"Start a new process with extra command line args"
+run,0,0,q,system "sleep 2",,,"Wait for process to start"
```

Comment:
torq actually has a function for this

```suggestion
run,0,0,q,.os.sleep 2,,,"Wait for process to start"
```

(same below on line 15 and anywhere else there's a sleep)